core/jazz_acl/swagger/swagger.json

```diff
@@ -0,0 +1,1596 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-01-30T23:30:18Z",
+    "title": "{api_deployment_node_title}",
+    "description": "API specification for ACL"
+  },
+  "host": "{api_host_name}",
+  "basePath": "/{domain}",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/{service_name}/services/{user}": {
```

Comment:
It might make more sense to take `user` should be a query param for this API. `/acl/services?user=$foo`

Reply:
updated